pkg/kallsyms/kallsyms_test.go

```diff
@@ -198,29 +237,147 @@ func TestRealKAllSymsParsing(t *testing.T) {
 	}
 }
 
-// BenchmarkKernelSymbolAddressJITParsing searches for the symbol while
-// reading the kallsyms file
-func BenchmarkKernelSymbolAddressJITParsing(b *testing.B) {
```

Comment:
This function is used for benchmarking (see `b *testing.B`). This function shouldn't be removed